### ISSUE TITLE
@cypher doesn't work in extend type Query

### ISSUE THREAD

**Issue by mohamedGamalAbuGalala** [neo4j-graphql/neo4j-graphql-js#395]
## Description

When you have such a big schema to design you might need to split your schema into multiple files instead of a single one.
We split the files to one base file that has all base (Query, Mutation, Subscription) types and all other files extend these types if they need.
The problem came when we need to extend the type `Query` and use `@cypher` inside it.

**The problem**
`@cypher` didn't work as expected in the `extend type Query` and only work on the root `Query` type

[Here](https://github.com/mohamedGamalAbuGalala/neo4j-graphql-js-bug) is a sample repo for this issue

**Sample Code**

```javascript
const typeDefs = `
type Query {
  rootQuery: String @cypher(statement: "WITH 'Wooorking 🐈 🐈 🐈 🐈' AS value RETURN value")
 }

 type Movie {
  title: String
  movieTestBase: String @cypher(statement: "WITH 'Wooorking 🐈 🐈 🐈 🐈' AS value RETURN value")
 }

 extend type Movie {
  movieTestExtend: String @cypher(statement: "WITH 'Wooorking 🐈 🐈 🐈 🐈' AS value RETURN value")
 }

 extend type Query {
  extendQuery: String @cypher(statement: "WITH 'NOT WORKING 😞 😞 😞 😞' AS value RETURN value")
 }
`
```

**Comment by benjamin-rood** [comment]
Workaround:

Use the `@graphql-toolkit/schema-merging` module, it will merge type extensions together, then you use print from graphql.js to export the typedefs back into an SDL string.

Which you can then pass back into `makeAugmentedSchema`.

**Comment by mohamedGamalAbuGalala** [comment]
Thanks, I'll try your solution :heart:

**Comment by johnymontana** [comment]
This should be fixed now (by #418) and we include this scenario in the [test here](https://github.com/neo4j-graphql/neo4j-graphql-js/blob/master/test/unit/augmentSchemaTest.test.js#L253-L257)

Please reopen if you're still seeing problems with this.
